backend/endpoints.py

```diff
@@ -0,0 +1,14 @@
+from flask import Flask, jsonify
+
+app = Flask(__name__)
+@app.route('/')
+def hello_world():
+    return 'Hello World'
+
+@app.route('/test', methods=['GET'])
+def test():
```

Comment:
added endpoint get method to make sure the test cases run smoothly, improves workflow